pkg/api/http.go

```diff
@@ -8,11 +8,12 @@ import (
 
 // HTTPError represents an error response from the GitHub API.
 type HTTPError struct {
-	StatusCode  int
-	RequestURL  *url.URL
-	Message     string
-	OAuthScopes string
-	Errors      []HTTPErrorItem
+	AcceptedOAuthScopes string
```

Comment:
Rather than exporting both `AcceptedOAuthScopes` and `OAuthScopes` does it make sense to export all the response headers?

Reply:
Yeah, maybe HTTPError could just expose all headers received from the response! Then integrators (including `cli/cli`) could wrap that